modules/gitops-release-notes-1-5-3.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-5-3_{context}"]
+= Release notes for {gitops-title} 1.5.3
+
+{gitops-title} 1.5.3 is now available on {product-title} 4.8, 4.9, and 4.10.
+
+[id="fixed-issues-1-5-3_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, all unpatched versions of Argo CD v1.0.0 and later were vulnerable to a cross-site scripting bug. As a result, an unauthorized user would be able to inject a javascript link in the UI. This issue is now fixed. link:https://bugzilla.redhat.com/show_bug.cgi?id=2096278[CVE-2022-31035]
+
+* Before this update, all versions of Argo CD v0.11.0 and later were vulnerable to multiple attacks when SSO login was initiated from the Argo CD CLI or the UI. This issue is now fixed. link:https://errata.engineering.redhat.com/docs/show/96528[CVE-2022-31034]
```

Comment:
Should we have a link to errata page or rather clustomer portal here? https://access.redhat.com/errata/RHSA-2022:5152